packages/migrate/src/index.ts

```diff
@@ -92,9 +93,9 @@ function runMigration(currentConfigPath: string, outputConfigPath: string): Prom
 
             diffOutput.forEach((diffLine: Change): void => {
                 if (diffLine.added) {
-                    process.stdout.write(green(`+ ${diffLine.value}`));
+                    logger.raw(green(`+ ${diffLine.value}`));
                 } else if (diffLine.removed) {
-                    process.stdout.write(red(`- ${diffLine.value}`));
+                    logger.raw(red(`- ${diffLine.value}`));
```

Comment:
Need test this manually, I am afraid here problem with newlines too

Reply:
<img width="1101" alt="Screenshot 2020-08-24 at 11 37 40 PM" src="https://user-images.githubusercontent.com/25279263/91080267-ecaa7900-e662-11ea-9ee5-420667f5a2dd.png">

Reply:
Extra newlines, I think keeping `process.stdout.write` is better here